front/app/utils/mapUtils/map.ts

```diff
@@ -14,6 +15,12 @@ import {
 
 export type LatLngTuple = [number, number, number?];
 
+export const projectPointToWebMercator = (geometry: __esri.Geometry) => {
```

Comment:
I'm not super familiar with the map projections so cool to see how this got fixed 👍